project/src/main/java/com/google/sps/servlets/Config.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
```

Comment:
Will this file be pushed to the repo if it contains an API key? Also, I believe the config.js file should be deleted since it definitely makes more sense to store the keys in the back end.

Reply:
I believe the file is still in .gitignore so you can only add it if you manually git add the file. Git won't prompt you to do so

Reply:
For these automatic Maven tests to pass, there must be a Config.java file, because Config.vairables are referenced throughout different files: https://github.com/googleinterns/voter-central/commit/dca35922db5e2c252ec7a19a7839f60dcdacf10f